### DISCUSSION TITLE
Week 1 Investment Summary

### DISCUSSION THREAD

**Discussion by luca2422** [kea-dpd/Group-Car-Recommendation#13] [Announcements]
Here is our week 1 investment summary below @lakruzz 

[Week 1 Investment summary .pdf](https://github.com/kea-dpd/Group-Car-Recommendation/files/12829934/Week.1.Investment.summary.pdf)

**Comment by lakruzz** [comment]
HI @kea-dpd/group-car-recommendation 

Thank you for your (although imaginary) invoice of €16 240 for a week's assistance by your team. That is a lot of money, so I sure am glad that you took the effort to give me an insight into your progress.

Overall I'm happy to see that your team made progress. I'm slightly worried if you team is truly cross-functional - can you ease my worries? Can you point me to an elaborated description of how you team is organized, who you are and how you plan to cover all bases on the competences needed - I'd like to keep this endeavour on budget, and I dot see any wiggle room to pull in any external resources.

I'm impressed by all the ground you covered in just one week. I had also hoped to find, just an oversight of what I might expect from next week? Do you have any core assumptions about the value the product brings (does it solve a need or a problem? and maybe even a plan to how to get in front of the audience, without having to rely on vanity numbers?

I hope to see e report like this next week, together with the next invoice, but I do hope that you'll get a chance to actually measure on how the intended customer segments react to your value offerings.

N.B. I really like you project/issue setup. I browsed around and I was happy to see that I have access to everything. Is there a short instruction to the process you work by, or how to read the project/issues data? I found an issue #5 that made me smile this is exactly what I needed - I hope you nail the Leaf Of Faith Assumptions - my fingers🤞

Feel free to mention me if you need my attention.

**Reply by EmilBystrup** [reply]
Hi @lakruzz 

We acknowledge your concerns regarding our team composition, but allow us to provide an explanation of why those worries may not be necessary.

Team Explanation: 
Even though most of us in the group are Business & IT students, with the exception of Zasia, who is pursuing multimedia design, our group dynamic operates seamlessly. We bring diverse academic interests and goals to the table, and it's this very diversity that strengthens our collaboration.

The vast and multifaceted domain of Business & IT studies has equipped us with a wide array of methods and tools. When we harness these collective skills and knowledge, we have at our disposal the comprehensive toolkit necessary to develop a fully realized and successful digital product. With the addition of Zasia, we believe that our team is perfect for this project.

Thankfully, we are all very adaptable. So, if something arises for which we don't have a solution, we possess the capabilities to resolve it. As a team of seven, we can also assign one or two team members to embark on a learning journey to address the issue, whether it involves a system, program, or any other challenge. In essence, should any unresolved issues arise, we are committed to acquiring the necessary skills and competencies.

This is our team and roles: 
Emil Schirmer Lenschau - Business Analyst & Organizer 
Magnus Christophersen – Programmer & Implementor
Mathias Villa Fonseca – Strategist & Conceptualiser
Jonas Hvid Nielsen - UX/QA, Conceptualiser, Technical stuff.
Lucas Molnar Jonasen - Quality assurance & Organizer 
Haidar Ansari - Technical stuff, Product development, Business Analyst
Zasia Meincke - Frontend, UX Design and research, So-Me marketing

Here's a picture of our team canvas: 
![image](https://github.com/kea-dpd/Group-Car-Recommendation/assets/99248174/3951b02b-fda3-4dc3-a5de-ca2036d17134)

In this weeks summary we will include an oversight of what you can expect from us the next week and go into further details on what we accomplished and why we did it. 

Best regards, 

Team Car Recommendation

**Reply by lakruzz** [reply]
Thank you - I like your approach. Very often large teams are not necessarily an advantage but it can be, if managed corrctly. And especially  thinking like this:

> _"...we can also assign one or two team members to embark on a learning journey to address the issue"_

Makes be happy - that is indeed a good utilization of many resources.

Keep up the good work 👍
